using_images/other_images/jenkins.adoc

```diff
@@ -386,14 +388,17 @@ Also note the `contrib` and `contrib/bin` subdirectories.  They allow for the in
 scripts for your image.
 
 The Jenkins image also provides auto-discovery and auto-configuration
-of slave images for the Kubernetes plug-ins by scanning the project Jenkins is deployed in for
-existing image streams with the label *role* set to *jenkins-slave*.
+of slave images for the 
+Kubernetes plug-in. The search for such images is done by examining existing image streams within the project that Jenkins is running in. The search specifically looks for image streams that have the label `role` set to `jenkins-slave`.
```

Comment:
"The search for such images is done by examining existing image streams within the project that Jenkins is running in."

is passive. Perhaps:

"The Jenkins image also provides auto-discovery and auto-configuration of the slave images for the Kubernetes plug-ins. The Jenkins image searches for these in the existing image streams within the project that it is running in."